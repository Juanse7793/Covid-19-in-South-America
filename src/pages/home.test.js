import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../Redux/configureStore';
import Home from './home';

describe('Home', () => {
  it('renders the Home Page and make a snapshot', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Home />
      </Provider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
