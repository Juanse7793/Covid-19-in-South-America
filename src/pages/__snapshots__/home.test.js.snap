// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the Home Page and make a snapshot 1`] = `
<DocumentFragment>
  <div
    class="Home"
  >
    <h1
      class="title"
    >
      Covid 19 in South American Countries
    </h1>
    <div
      class="first-card-cont"
    >
      <img
        alt="South american countries"
        class="sac-img"
        src="Southamerica.svg"
      />
      <div
        class="first-card"
      >
        <h2
          class="subtitle"
        >
          South America
        </h2>
        <p
          class="description"
        >
          442.5M hab
        </p>
        <button
          class="btn-about"
          type="button"
        >
          About
        </button>
      </div>
    </div>
    <div
      class="second-card-cont"
    >
      <h2
        class="stats"
      >
        STATS BY COUNTRY
      </h2>
      <div
        class="second-card"
      />
    </div>
  </div>
</DocumentFragment>
`;
